server/action_creators.js

```diff
@@ -80,6 +80,17 @@ export function addEntriesToOrder(orderId, newEntries) {
   };
 }
 
+export function setReadyForBill(orderId) {
+  return (dispatch, getState) => {
+    return Order.findOneAndUpdate({ id: orderId }, { status: constants.READY_FOR_BILL })
+    .then(() => {
+      return Order.getOrders();
```

Comment:
So this is the pattern I wanted to talk to you about. Should we be making two database calls here? In a couple of the other actions, we incrementally update state for only the order that got changed / added, instead of fetching and replacing all of them. This way is definitely simpler but adds a database call per action.